libraries/EEPROM/src/EEPROM.cpp

```diff
@@ -66,31 +63,59 @@ EEPROMClass::~EEPROMClass() {
 }
 
 bool EEPROMClass::begin(size_t size) {
-  if (size <= 0) {
-    return false;
+  if (size <= 0) return false;
```

Comment:
please try to not use one liners like this one :) new line and brackets are more readable and conform to IDF guidelines as well